include/filters-sanitization.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Setup specific filters useful for sanitization.
+ *
+ * Extract from PLL_Admin_Filters to be able to use in a REST API context.
+ *
+ * @since 2.9
+ */
+class PLL_Filters_Sanitization {
+	/**
+	 * Language used for the sanitization depending on the context.
+	 *
+	 * @var string $locale
+	 */
+	public $locale;
+
+	/**
+	 * Instance of PLL_Model
+	 *
+	 * @var PLL_Model
+	 */
+	protected $model;
+
+	/**
+	 * Constructor: setups filters and actions.
+	 *
+	 * @since 2.9
+	 *
+	 * @param object $polylang
+	 * @param string $locale Locale code of the language.
+	 */
+	public function __construct( &$polylang, $locale ) {
+		$this->model  = &$polylang->model;
+		$this->locale = $locale;
+
+		// We need specific filters for some languages like German and Danish
+		$specific_locales = array( 'da_DK', 'de_DE', 'de_DE_formal', 'de_CH', 'de_CH_informal', 'ca', 'sr_RS', 'bs_BA' );
+		if ( array_intersect( $this->model->get_languages_list( array( 'fields' => 'locale' ) ), $specific_locales ) ) {
```

Comment:
Now that we pass `$locale` to this class, we should not need all languages anymore. So I suppose that we can just test that `$locale` is in the array `$specific_locales`. This should also simplify the class (no more need to pass `$polylang` and use the `PLL_Model` instance.

Reply:
Yes! Good Idea, I'm going to implement it.

Reply:
Done !